### ISSUE TITLE
Add Packgage.JSON and webpack.config.js to the 3 WooCommerce Admin Extension Examples

### ISSUE THREAD

**Issue by spraveenitpro** [woocommerce/woocommerce-admin#2620]
**Describe the bug**

There are 3 example extensions [available here](https://github.com/woocommerce/woocommerce-admin/tree/master/docs/examples/extensions) to quickly scaffold but they are not usable out of the box as they do not have the `package.json` and the `webpack.config.js` with in them.

**To Reproduce**

0. Clone WooCommerce-admin to your plugins folder at directory `\WordPress_installation_dir\wp-content\plugins\`
1. Install dependencies, if you haven't already with `npm install` 
2. Goto directory `\WordPress_installation_dir\wp-content\plugins\woocommerce-admin`
3. Build the example extension by running the npm script and passing the example name: `npm run example -- --ext=<example>`
4. Activate the newly created plugin by going to `Plugins >> Installed plugins`
5. Now goto directory `\WordPress_installation_dir\wp-content\plugins\example-extension` and run `npm start` to see the error saying a lack of `package.json`

**Expected behavior**
To be able to start NPM from the example extension folder and then make changes to the code to see the changes be rendered in real time in the browser.

**Additional Context:**

Basically, this tutorial does not work: https://github.com/woocommerce/woocommerce-admin/tree/master/docs/examples/extensions

**Comment by psealock** [comment]
>Now goto directory \WordPress_installation_dir\wp-content\plugins\example-extension and run npm start to see the error saying a lack of package.json

`npm start` should be run from root of this repo. You can also edit the example in this repo as well.
